mandatory/thinking-like-a-programmer/task.md

```diff
@@ -29,12 +29,18 @@ After you've watched these videos I'd like you to answer these questions
 
 ## 1. What do you think the most important quality for a programmer is?
 
-<!-- Write your answer here -->
+I suppose before af all programer must first be an optimistic person, even when you facing very difficult problems, he/she must be in the mode that he/she can solve it today and tomorrow. The presence of logic thinking ability shows the programmer how qualified he/she is.
+Programmer a person who can use time well and works organized.
 
 ## 2. When trying to solve a challenge, what should you do first?
 
-<!-- Write your answer here -->
+I want to share my own experiences. When I am doing project and it didn't give what I expect or it had a bug,
+-first thing I am doing relaxing just relax don't panic,
+-after try to read code step by step, console log it to see what is happening on each step,
 
 ## 3. What should you do if you get stuck?
 
-<!-- Write your answer here -->
+-and then nothing could find google is your best friend, trying to find same errors as mine, stackoverflow.com is the my favorite site which I go first,or anything else
+-if you have friend or colleague who might help with your problem then ask them
+-if haven't solve your problem then make some coffee or tea or just give a rest, or if it is evening time sleep well and take a look tomorrow with fresh mind it will look much more different then now
+-but if don't have time and you have lots of works then do another work and try problem solving after!
```

Comment:
Really nice you added other concepts aside from the ones that were in the links. :)